AsyncDisplayKit/ASMapNode.mm

```diff
@@ -265,17 +277,23 @@ - (void)addLiveMap
     [_mapView addAnnotations:_annotations];
     [weakSelf setNeedsLayout];
     [weakSelf.view addSubview:_mapView];
-    
-    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
-      [_mapView setCenterCoordinate:_centerCoordinateOfMap];
+
+    if (self.showAnnotationsOptions & ASMapNodeShowAnnotationsOptionsZoomed) {
+      BOOL const animated = self.showAnnotationsOptions & ASMapNodeShowAnnotationsOptionsAnimated;
+      [_mapView showAnnotations:_mapView.annotations animated:animated];
     }
+    
+//    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
+//      [_mapView setCenterCoordinate:_centerCoordinateOfMap];
+//    }
   }
 }
 
 - (void)removeLiveMap
 {
+  self.userInteractionEnabled = false;
```

Comment:
We may not want to make this change. Since ASMapNode inherits from ASControlNode, the user may very well have attached handlers for control events e.g. tap on the map to open a larger map. Let's instead have ASMapNode default to `userInteractionEnabled = YES` (ASControlNode defaults to NO which is something we may want to change in the future) and let the user change it from there if they want.